tensorflow/core/kernels/slice_op_cpu_impl_6.cc

```diff
@@ -1,18 +0,0 @@
-/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Do you means these?
file: tensorflow/core/kernels/slice_op.h
```C++
template <typename Device, typename T, int NDIMS>
struct Slice {
  void operator()(const Device& d, Tensor* out, const Tensor& in,
                  const gtl::ArraySlice<int64>& slice_indices,
                  const gtl::ArraySlice<int64>& slice_sizes) {
    switch (in.dims()) {
      case NDIMS:
        internal::SliceUsingEigen<Device, T, NDIMS>(d, out, in, slice_indices, slice_sizes);
        break;
      default:
        if (Eigen::internal::is_same<Device, Eigen::GpuDevice>::value) {
          internal::SliceSimpleGpu<Device, T>(d, out, in, slice_indices);
        } else {
          internal::SliceSimple<Device, T>(d, out, in, slice_indices);
        }
        break;
    }
  }
};
```
and file: tensorflow/core/kernels/slice_op_cpu_impl_1.cc
```C++
#define CPU_PROVIDED_IXDIM 1
#include "tensorflow/core/kernels/slice_op_cpu_impl.h"
#undef CPU_PROVIDED_IXDIM
```